examples/terraform/bootkube-install/README.md

```diff
@@ -35,8 +35,20 @@ matchbox_rpc_endpoint = "matchbox.example.com:8081"
 ssh_authorized_key = "ADD ME"
 ```
 
+##### Self-hosted etcd
+
 You may set `experimental_self_hosted_etcd = "true"` to deploy "self-hosted" etcd atop Kubernetes instead of running etcd on hosts directly. Warning, this is experimental and potentially dangerous.
 
+##### OEM Customisation
+
+You may set `container_linux_oem = "<oem_id>"` to use a specific oem version as installation base. For example setting `container_linux_oem = "vmware_raw"` will use an installation image (as base) with included Guest Tools.
+
+Popular OEM images: `ami`, `cloudstack`, `digitalocean`, `packet`, `rackspace`, `vmware_raw` and `xen`.
```

Comment:
Bare-Metal is the primary use case and the default "" option is suitable. I see `vmware_raw`, `packet`, and `xen` as plausible options since users can treat those platforms like bare-metal and PXE boot machines there.

The use case for cloud images like `ami` and `digitalocean` seems more nebulous. Can you elaborate on your setup?

Also, let's move this down to the bottom. I don't want to confuse users, most of whom shouldn't need to set this.